core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/view/ClusterApiViewToClusterViewV4ResponseConverter.java

```diff
@@ -31,7 +29,7 @@ public ClusterViewV4Response convert(ClusterApiView source) {
         clusterViewResponse.setStatus(source.getStatus());
         clusterViewResponse.setHostGroups(convertHostGroupsToJson(source.getHostGroups()));
         clusterViewResponse.setCertExpirationState(source.getCertExpirationState());
-        addSharedServiceResponse(source, clusterViewResponse);
+        datalakeService.addSharedServiceResponse(source, clusterViewResponse);
```

Comment:
this will be very slow when we list clusters

Reply:
we discussed it